From a3d405e9d797fe7f7a5fbd715b69e46bce87728a Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 6 Jul 2022 11:03:32 +0200
Subject: [PATCH] arm64: defconfig: enable support of the STM32 spi controller

Add compilation of CONFIG_SPI_STM32 as a module in
fragment-02-defconfig-addons

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: I81c4420940eb2073c81fa13dba06782540361632
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index b6a7b2607428..de5dd55bd81b 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -14,6 +14,7 @@ CONFIG_PWM_STM32_LP=m
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REMOTEPROC=y
 CONFIG_RTC_DRV_STM32=y
+CONFIG_SPI_STM32=m
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
 CONFIG_STM32_DMA3=y
-- 
2.39.5

